src/Numerics/Mesh/Filters.jl

```diff
@@ -212,7 +212,7 @@ function χ(x)
     if (x == 0.5)
         return 1
     else
-        return sqrt(-log(1 - 4 * (abs(x) - 0.5)^2) / 4 * (abs(x) - 0.5)^2)
+        return sqrt(-log(1 - 4 * (abs(x) - 0.5)^2) / (4 * (abs(x) - 0.5)^2))
```

Comment:
Probably should not be `0.5` by `FT(1/2)` or 
```suggestion
        return sqrt(-log(1 - (2abs(x) - 1)^2) / ((2abs(x) - 1)^2))
```

Reply:
(otherwise you won't be respecting the type of `x`)

Reply:
@jkozdon you're right I'll adjust this accordingly